server/api/elections.py

```diff
@@ -16,6 +16,12 @@
             "type": "string",
             "enum": [audit_type.value for audit_type in AuditType],
         },
+        "ballotPollingType": {
+            "type": "string",
+            "enum": [
+                ballot_polling_type.value for ballot_polling_type in BallotPollingType
+            ],
+        },
```

Comment:
we should also add some validation in `validate_election` that `ballotPollingType` is always set for ballot polling audits and never set for other audit types